contributing/documentation/style-guide/images.md

```diff
@@ -0,0 +1,70 @@
+---
+description: >-
+  Learn how to add images and its folder structure.
+---
+
+# Images
+
+Images should be stored in an `images` or `assets` directory next to the `.md` file referencing them, using relative paths. Use descriptive alt text and captions to ensure accessibility and Search Engine Optimization (SEO) benefits. Optimize images for the web to improve page load times.
+
+## General Guidelines
+
+* Maximum image width: **800px**.
+
+* Supported formats: `png`, `jpg`, `gif` (use the most efficient compression available).
+
+* Prohibited formats: `bmp`, `tiff`, `swf` (Flash)
+
+## Folder Structure
+
+If images are used, these must be stored in an `images` or `assets` directory next to the `.md` file referencing them using relative paths.
+
+Maintain a clear and consistent directory structure:
+
+```plaintext
+/topic               # Main documentation directory
+│── README.md        # Main content file
+│── another-page.md  # Another markdown file
+│── /images          # Image storage
+│   ├── image1.png
+│   ├── image2.jpg
+│
+└── /subtopic        # Subdirectory for related content
+    │── README.md
+    │── topic.md
+    │── another-topic.md  
+    │── /images      # Subdirectory for subtopic images
+        ├── image3.png
+```
+
+## Adding images in Markdown
+
+Use the following markdown syntax to insert images:
+
+* Image with Caption
+
+```markdown
+![Caption](../images/sample.png)
+```
+
+* Image with Alt Text (Markdown/GitHub):
+
+```markdown
+![Alt text describing the image](images/img.png)
+```
+
+* Image with Alt Text (GitBook)
```

Comment:
I think some more explanation is needed here.
The rest of the article talks about markdown, but this example is using HTML tags.

It would be good with a couple of words around this is what the syntax looks like when images have been added through the platform where the documentation is hosted. And then perhaps also something about how we still recommend using the markdown approach when adding them via GitHub? 🤔

Reply:
Makes sense. I have added some text around it. Let me know what you think🤔